src/mappings/Controller/mapping.ts

```diff
@@ -88,6 +88,34 @@ function unregisterScheme(avatarAddress: Address, scheme: Address): void {
   let controllerScheme = ControllerScheme.load(crypto.keccak256(concat(avatarAddress, scheme)).toHex());
   if (controllerScheme != null) {
     controllerScheme.isRegistered = false;
+    let dao = DAO.load(avatarAddress.toHex());
+    if (dao != null) {
+      dao.numberOfQueuedProposals = dao.numberOfQueuedProposals.minus(
```

Comment:
this cannot be negative because of https://github.com/daostack/arc/pull/802 . right ?

Reply:
No this happens when a scheme is unregistered and still have proposals on it